vpd-manager/manager.cpp

```diff
@@ -560,6 +560,14 @@ void Manager::performVPDRecollection()
             singleFru["inventoryPath"]
                 .get_ref<const nlohmann::json::string_t&>();
 
+        inventory::InterfaceMap interfacesPropMap;
+        clearVpdOnRemoval(INVENTORY_PATH + inventoryPath, interfacesPropMap);
```

Comment:
Will this clear all the FRUs data in the whole system -or- only under the groupEEPROM[0] interface path?

Reply:
It will only clear VPD for the inventory path being sent to the function.

Reply:
So this is the single interface which needs to be cleared?